source/user-guide/faq.rst

```diff
@@ -26,7 +26,7 @@ Make it executable and double-click it.
 |question| How can I integrate AppImages with the system?
 ---------------------------------------------------------
 
-Using the optional appimaged daemon, you can easily integrate AppImages with the system. The daemon puts AppImages into the menus, registers MIME types, icons, all on the fly. You can download it from this repository. But it is entirely optional.
+Using the optional appimaged daemon, you can easily integrate AppImages with the system. The daemon puts AppImages into the menus, registers MIME types, icons, all on the fly. You can download it from `this <https://github.com/probonopd/go-appimage/releases/tag/continuous>`_ repository. But it is entirely optional.
```

Comment:
This information is clearly outdated. Instead of changing this link, you should link to the corresponding page.

https://docs.appimage.org/user-guide/run-appimages.html#integrating-appimages-into-the-desktop

It discusses various alternatives. The link _there_ has been fixed. The link in https://docs.appimage.org/introduction/software-overview.html#appimaged has been fixed as well.

For the record, appimaged on https://docs.appimage.org/introduction/software-overview.html#appimaged should be moved to "third-party tools" nowadays. It is just as third party as linuxdeployqt or AppImageLauncher are.